core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/plugins/ai-content-prompt.plugin.ts

```diff
@@ -148,15 +148,15 @@ export class AIContentPromptView {
                 filter((deleteContent) => deleteContent)
             )
             .subscribe(() => {
-                const nodeInformation: DotTiptapNodeInformation = findNodeByType(
+                const nodeInformation: DotTiptapNodeInformation[] = findNodeByType(
```

Comment:
do you need the array? I see that you are always accessing the first index, should we destructure the array here or access the first index here right away?

Reply:
I changed from single to multiple, just to make the function more generic. and eventually re use it for the image-placeholder search.

Reply:
done